browser/misc_metrics/sources.gni

```diff
@@ -0,0 +1,17 @@
+brave_browser_misc_metrics_sources = [
+  "//brave/browser/misc_metrics/page_metrics_service_factory.cc",
+  "//brave/browser/misc_metrics/page_metrics_service_factory.h",
+  "//brave/browser/misc_metrics/page_metrics_tab_helper.cc",
+  "//brave/browser/misc_metrics/page_metrics_tab_helper.h",
+]
+
+brave_browser_misc_metrics_deps = [
+  "//brave/browser:browser_process",
+  "//brave/browser/profiles:util",
+  "//brave/components/misc_metrics",
+  "//chrome/browser:browser_process",
```

Comment:
just noticed that there is a bunch of unneeded deps here. We need to drop them and convert this file to BUILD.gn instead (since sources.gni format is used only when needed, i.e. there is a circular dep like //chrome/browser

Reply:
any ideas on where to place the `//brave/browser/misc_metrics` dep? it seems that most `browser` level modules use sources.gni, so i'm having a tough time finding an example of this. Also, can you clarify which deps on unneeded? it seems that I need quite a few of them for the keyed service factory

Reply:
hmm, not sure what i was looking for.. Apparently all of these are needed for the factory, sorry for the wrong comment!